libs/api/domains/health-insurance/src/lib/graphql/scalars.ts

```diff
@@ -0,0 +1,14 @@
+import { registerEnumType } from '@nestjs/graphql'
+import { HealthInsuranceAccidentNotificationAttachmentTypes, HealthInsuranceAccidentNotificationConfirmationTypes, HealthInsuranceAccidentNotificationStatusTypes } from './../types'
+
+registerEnumType(HealthInsuranceAccidentNotificationConfirmationTypes, {
+  name: 'HealthInsuranceAccidentNotificationConfirmationTypes',
+})
+
+registerEnumType(HealthInsuranceAccidentNotificationStatusTypes, {
+  name: 'HealthInsuranceAccidentNotificationStatusTypes',
+})
+
+registerEnumType(HealthInsuranceAccidentNotificationAttachmentTypes, {
+  name: 'HealthInsuranceAccidentNotificationAttachmentTypes',
+})
```

Comment:
Fine to have a simpler name in TypeScript and just register the long name in a GraphQL decorator and/or registerEnumType. :)